Street Trees/STConfirmationPageViewController.swift

```diff
@@ -0,0 +1,168 @@
+//
+//  STConfirmationPageViewController.swift
+//  Street Trees
+//
+//  Created by Pedro Trujillo on 10/17/16.
+//  Copyright © 2016 Code for Orlando. All rights reserved.
+//
+
+import UIKit
+import StreetTreesTransportKit
+
+
+//**********************************************************************************************************************
+// MARK: - Protocol
+
+protocol STConfirmationPageViewControllerDelegate: NSObjectProtocol {
+    func ConfirmationFormViewController(form: STConfirmationPageViewController, didCompleteWithWufooForm anWufooForm: STTKWufooForm)
+}
+
+//**********************************************************************************************************************
+
+class STConfirmationPageViewController: UIViewController, UITableViewDataSource, UITableViewDelegate
+{
+    
+    //var aTree:STTKStreetTree?
+    var anAddress: STTKStreetAddress?
+    var aContact: STTKContact?
+    
+    //just initial test values for vars
+    var treeName = "Example Tree"
+    var treeImage : UIImage?
+    var address = "101 Example Ave"
+    var cityStateZip = "Orlando, Fl 32801"
+    var name = "Bob Bobington"
+    var phone = "407-555-5550"
+    var email = "bob@google.com"
+    
+    weak var  delegate:STConfirmationPageViewControllerDelegate?
+    
+
+    override func viewDidLoad()
+    {
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
+        
+        
+    // MARK: Get information from Wufoo Form:
+        
+        //treeName = (aTree?.name)!
+        //treeImage =  aTree.
+        address = (anAddress?.streetAddress)!  + " " +  (anAddress?.secondaryAddress)!
+        cityStateZip = "\(anAddress?.city), \(anAddress?.state) \(String((anAddress?.zipCode)!))"
+        name = (aContact?.name)!
+        phone = (aContact?.phoneNumber)!
+        email = (aContact?.email)!
+        
+    }
+
+    override func didReceiveMemoryWarning()
+    {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+    
+
+    /*
+    // MARK: - Navigation
+
+    // In a storyboard-based application, you will often want to do a little preparation before navigation
+    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
+        // Get the new view controller using segue.destinationViewController.
+        // Pass the selected object to the new view controller.
+    }
+    */
+    
+    // MARK: - Table View Methods:
+    
+    func numberOfSectionsInTableView(tableView: UITableView) -> Int
+    {
+        return 1
```

Comment:
Create an enum for the cells and then create a datasource property or function that returns a data source struct that contains your content for the tableview. You should be able to query your new struct for the amount of sections and query for how many items are in a section.